src/test/shell/bazel/bazel_toolchain_test.sh

```diff
@@ -22,8 +22,8 @@ CURRENT_DIR="$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)"
 source "${CURRENT_DIR}/../integration_test_setup.sh" \
   || { echo "integration_test_setup.sh not found!" >&2; exit 1; }
 
-if [ "${PLATFORM-}" = "darwin" ]; then
-  echo "Skipping test: linaro toolchain is not supported on darwin host."
+if [ "${PLATFORM-}" = "darwin" -o "${MACHINE_TYPE}" = "ppc64le" ]; then
+  echo "Skipping test: linaro toolchain is not supported on darwin host or ppc64le Linux."
```

Comment:
Yes I mean this test is made to work only on Linux x86_64 so it's better to test that we are in the good architecture rather than list all the platform it cannot work on.